reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -22,295 +22,284 @@ package com.here.ort.reporter.reporters
 import ch.frankel.slf4k.*
 
 import com.here.ort.model.OrtResult
-import com.here.ort.model.VcsInfo
+import com.here.ort.model.Project
 import com.here.ort.model.config.CopyrightGarbage
-import com.here.ort.model.config.ScopeExclude
 import com.here.ort.reporter.Reporter
 import com.here.ort.reporter.ResolutionProvider
 import com.here.ort.reporter.reporters.ReportTableModel.ErrorTable
 import com.here.ort.reporter.reporters.ReportTableModel.ProjectTable
 import com.here.ort.reporter.reporters.ReportTableModel.ResolvableIssue
-
 import com.here.ort.utils.isValidUrl
 import com.here.ort.utils.log
 
 import java.io.File
-import java.util.SortedMap
+
+import javax.xml.parsers.DocumentBuilderFactory
+
+import kotlinx.html.*
+import kotlinx.html.dom.*
 
 class StaticHtmlReporter : Reporter() {
+
+    private val css = """
+
```

Comment:
Is this blank line intended?

Reply:
Yes, otherwise there is no line break between `<style>` and `body {` in the result.